src/index.html

```diff
@@ -7,6 +7,351 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
```

Comment:
IHO, is a little better because we can see the content without reading the whole line
```suggestion
            <a href="#apple" class="nav__link is-active">
              apple
            </a>
```